### ISSUE TITLE
Overview of discord proposals 

### ISSUE THREAD

**Issue by itizice** [Zer0-Foundation/zer0hub#1]
**GOVERNANCE**
-zer0 focus group coordinates proposals
-zer0 focus group can implement minor proposals 
-mockjup holders get to vote on major proposals 

**FOCUS APP**
-Modify the focus app a bit for longer tem projects 
-Allow ideas/tasks/projects to be proposed and appropriately triaged
-coordination of sub-groups

**BURN N MINT**
-Burn and mint "zer0" eventually 

**JUPITER INTEGRATION**
-Make sure we stay at least Jupiter adjacent, we want to stay with stepdad jup.
-MockJup should have some utility in Júpiter 
-Future testing on other JUP projects down the line? websites launches etc

**ZEROHUB**
-Meme repository in the style of 9gag 
    - sign in with wallet or socials and to upload the memes and etc.. 
-Vote for meme competitions, confessions, Zer0Awards, etc.
     -Vote for the winner and get a portion of the award
-Zerob0i confessionals
     - An anonymous way to tell your story
          -Scammed, rugged, lost it all, extreme bag holding 
-Annual? Zer0Awards. The IG Nobel prizeof crypto. Worst rugs, worst trades, worst predictions, best shitcoin revival, ECT...

**DISCORD**
-Hold 50k for 30 days for role
-Burn 100k for permanent role
-stickers and emojis for mockjup
-mockJUP stay in the degen category?
-create our own categories?

**CHALLENGE/GIVEAWAY** 
-Stake MockJup in wallet for mental health donations 
-48 hour no-check mockJUP, 7-day challenge next.
-Picture challenge game
-badge for Twitter accounts for mockJUP owners and have giveaways
-mockjup faucet address for mjup twitter, so everyone that stops by can have a lil mjup
-Community hero award 
     - propose to set aside some mockJUP funds for the champions among us
-AMA to the group on spaces W/ meow to introduce governance/future
- Bug bounty for zerobois focus app
     - steal any mockJUPs "'staked" can give a bounty of 250k mockJUP
-Mockjup up 100% on exclusive access to Julian onlyfans for holders

**MERCH**
- meow to fund first 100 merch drops for Zer0 focus group
- merch store with anonymous drop shipping on zerohub
-a mockJUP plush? Shirt? Commemeritive piss jug?

**ZERO AWARDS / NFT**
- free to claim NFT for holders above 10k/100k or more
     -Burn for higher your chances for a legendary/mythic.
- pay for mint in mJ burn portion &/or allocate a portion to community treasury
- pfp nft collection made by us and for the zero community
-Fractionalize Manifest0, airdrop to mJ holders

--------------
Sources
--------------

**GOVERNANCE**
NFT_Unlimited 8d
mockjup holder get to vote on proposal

Julian zer0 Today
people can "stake" and back ideas and make it happen by date or lose their stake etc. dunno
just floating it as an idea TastyJelly Today at 8:12 AM Oh that is decentralized yes, stake and get votes Julian zer0 Today at 8:12 AM hahaha
if nobody uses it it's fine also

**FOCUS APP**
Julian zer0 Today 
modify the focus app a bit for longer ranged ideas and to allow ideas to float up and coordination of sub-groups. 

**BURN N MINT**
curselol - zer0 02/12/2024 
When fork to "zero" 

deltaAcrowface 02/13/2024 
But creating a new token called zero is one way of potentially sending "'mockjup to zero" lol

**JUPITER INTEGRATION**
SolSmallBrain zer0 02/12/2024
so what if mockjup is used for memecoins launchpad

Trujilloa o2/13/2024 
MockJup should have some utility in Júpiter 

davetheshrew zerO 02/14/2024 
we could be used for future testing on other JUP projects down the line? websites launches etc, test stuff (edited)

**ZEROHUB**
5inco Yesterday 
"zero hub" for memes in the style of 9gag 
 sign in and to create your profile to upload the memes and etc.. with Sol wallet or is better to go web2 email?
@Julianzer0 
can have sign-in with Solana as well as sign-in with socials think... 
5inco
been thinking maybe if we use Sol wallets, we can have a pot for weekly upvote meme with Mockjup or something like that (edited)

enflux Yesterday 
zero hub that offers more ways to spend your mockjup

**COMMS**
aldasoro.zeta 02/13/2024 
What about twitter raids giving away mockjup so they can send it to the incinerator??

**DISCORD**
Julianzer0 02/13/2024 
 having 50k mockJUP signifies some degree of shared commitment/sta ke in the values/culture here. Will also consider if you have less than 50k but have been active/OG since the start and basically that's what was you started w/ and haven't so ld since etc. (or maybe sold< 20% etc.) But yeah am flexible on this tbh as circumstances can be different for everyone.. it's more about having the cultural fit more than anything.
03

Alcogelso 02/14/2024 
we need some stickers and emojis for mockjup

yosachain 
Why is mockJUP in the degen category? can we create our own categories?

**CHALLENGE** 
Julian zer0
I'm going to do a 48 hour no-check mockJUP price challenge. LOL who's with me?
succeed in this one l'll do a 7-day challenge next.

**BURN**
@Julian |zer0
 if anyone wants to skip the 30 day waiting/ queue and want a lifetime role, can burn 10Ok mockJUP

wakander 02/14/2024 
it would be cool to make a badge for Twitter accounts for mockJUP owners and have weekly giveaways for these badges fora month

**GIVEAWAY**
davetheshrew zer0 02/09/2024 
we should all make a mockjup faucet address also and put it somewhere like top of mjup twitter, so everyone that stops by can have a lil mjup

Apeshi el doggo 6d
Stake MockJup in wallet for mental health problems 

Martin Yesterday 
Julian is our hero - propose to set aside some mockJUP funds for him at some point 3

brbsol | zerO Today 
AMA to the group on spaces
Force all u shy zerobois up with free mjup

Julian zer0 9h
 hack this site and steal any mockJUPs "'staked" can give a bounty of 250k mockJUP

GizmoDroid zer0 02/12/2024
Mockjup up 100% on exclusive access to Julian onlyfans for holders

**MERCH**
Apeshiel doggo o2/02/2024
Should seriously get meow to do a run of some shirts.

judeplus 02/05/2024 
i would love a mockJUP plush

**NFT**
@Martin 02/08/2024
 with mockJup - free to claim NFT for holders above 10k/100k or more

kankerS| zerO 02/09/2024
Mint in $mockJUP could raise a treasury.  theghost 02/09/2024
And burn it

AteHash zer0 02/11/2024
What if we create a pfp nft collection made by us and for the zero community, paying mockJUP and then burn it

rebound 02/11/2024
Mockjup nfts that can only be purchased with Mockjup that will be burned.

kankerS |zer0 02/13/2024 
build a treasury for the DAO by minting nfts in $mockJUP. 
Guac did this and raised like 6% of totaly supply.

rebound Yesterday 
launch an NFT project for Mockjup. It can then be fractionally traded with Mockjup/mockjup nfts like ovols has done. 

enflux Yesterday at
A pfp project for mockjup
Burn to mint the higher your chances for a legendary/mythic.
Fractionalized manifesto fungible token which was dropped to mockjup holders

enflux Yesterday 
mockjup used to mint goes back into a pool that can be earned back through staking your Oboi nft (edited) 

But, how much supply of nfts? 
How much to burn?
What type of art?

**Comment by ZeroFeet** [comment]
the merch could be key esp if meow were to wear it in an interview

**Comment by aricryptogold** [comment]
Key thing we need to get going is a Communication channel and start announcing that we are doing work

if Meow could retweet some of our major milestones that would be very powerful.....first one coming, the Manifesto (he's retweeting Wen stuff already)

**Comment by itizice** [comment]
> Key thing we need to get going is a Communication channel and start announcing that we are doing work
> 
> if Meow could retweet some of our major milestones that would be very powerful.....first one coming, the Manifesto (he's retweeting Wen stuff already)

Definitely, so far the MJ narrative has been driven by Meow. Time for us to show initiative and drive our own narrative. In exactly the same way ovoals has. 

To me, Manifest0 is an obvious short term goal.    Get that out the door in a sensible time with a decent consensus and we know we're on the right path.

**Comment by ZeroFeet** [comment]
*will we fund 100 merch drops?..or Merch drop for the 100? I think we mean the second one but as currently phrased a bit confusing.

**Comment by itizice** [comment]
> How about this? That is accurate, right? We still capped at 100 roles?

'Meow agreed to personally fund merch drop for the 100 Zer0 role holders"

**Comment by aricryptogold** [comment]
> > How about this? That is accurate, right? We still capped at 100 roles?
> 
> 'Meow agreed to personally fund merch drop for the 100 Zer0 role holders"

that's what he said on the voicecall
